Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
```

Comment:
On the other hand, maybe it is simpler in the short term to grab the url from the .kbase_config (or .authrc, I forget the name. We should use the same name as the py client (which is embedded in baseclient.py).